deployments/daemon-set/nginx-ingress.yaml

```diff
@@ -29,6 +29,13 @@ spec:
           hostPort: 443
        #- name: prometheus
          #containerPort: 9113
+       #- name: readiness-port
+         #containerPort: 8081
+        #readinessProbe:
```

Comment:
we comment out the fields in the YAML by adding `#` in front of the name of a field. Note that the first letter of the field is in its correct position.  The readinessProbe field doesn't follow that convention. Could you fix that?

(before fixing, please see the suggestion about enabling readiness by default)